app/scripts/components/boilerplate/open-data-nav.jsx

```diff
@@ -9,7 +9,7 @@ const OpenDataNav = React.createClass({
     return (
       <ul className="open-data-nav">
         <li>
-          <Link activeClassName="active" to="/">
+          <Link activeClassName="active" to="/homepage/">
```

Comment:
If we update the `<Router>` in app.jsx to an IndexRoute, this can go back to just "/"